lib/tasks/storage/storage.rb

```diff
@@ -149,10 +152,20 @@ def not_a_mirror
     "correct."
   end
 
-  def primary_service
+  def mirror_primary_not_disk_service
+    "#{prefix}: Mirror primary service is not a disk service, ensure " \
+    "config/storage.yml is correct."
```

Comment:
Ought to be more specific about what "correct" means in this context.

Need to rephrase, but something like?

> Mirror primary service is not an on-disk service. Records are only pushed to off-site storage after 7 days because {REASONS} and only then do we promote the secondary mirror to be the source of truth and delete the on-disk file.

Reply:
Am going to link to a page one configuring storage on alaveteli.org

See #6796